x/validator-preference/types/params.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+// ParamKeyTable the param key table for launch module
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+// NewParams creates a new Params instance
+func NewParams() Params {
```

Comment:
are params tbd in future PRs as well?

Reply:
tbh haven't really needed to use params yet. This was just a boilerplate copy paste